js/shared.js

```diff
@@ -378,4 +377,75 @@ async function getAllWitnessesOfSource(source){
  */ 
 function undoBrowserSelection(s){
     s?.removeAllRanges?.() ?? s?.empty?.()
+}
+
+/** 
+ * Confirm the shelfmark's inclusion in "GoG-manuscripts" dynamic collection. 
+ * @see {@link findShelfmark #113} to verify shelfmark is appropriate. 
+ *
+ * @async
+ * @function addManuscriptToGoG
+ * @param { string } shelfmark Identifier to include.
+ */ 
+async function addManuscriptToGoG(shelfmark) {
+    try {
+        /** Wash shelfmark by 
+         * Removing specific special characters @ $ % * ?
+         * Replace multiple spaces with a single space
+         * Removing trailing or leading whitespace
+         * */
+        const cleanShelfmark = shelfmark.replace(/[@$%*?]+/g, '') 
+        .replace(/\s+/g, ' ') 
+        .trim() 
+
+        // check for existing annotation with cleaned shelfmark
+        const query = {
+            "target": `${cleanShelfmark}`,
```

Comment:
You don't need the string literal syntax here, just `"target":cleanShelfmark`.

Ehh we do not want false positives from other apps in this query, especially if you are reducing this to a single property match.  Use `"__rerum.generatedBy" : __constants.generator` to make sure it only cares about matching on data generated by GoG and that will futureproof it.